### DISCUSSION TITLE
Date on Immich server incorrect - stuck on UTC

### DISCUSSION THREAD

**Discussion by drelephant** [immich-app/immich#12624] [Q&A]
### The bug

I'm running Immich via docker compose after following the instructions from https://immich.app/docs/install/docker-compose/.

My timezone is UTC+10, AEST.

When I take photos, they show as being taken "tomorrow"!!

It seems that the server takes the AEST timezone, but incorrectly uses the UTC time.

### The OS that Immich Server is running on

Ubuntu 20.04.6 LTS

### Version of Immich Server

v1.115.0

### Version of Immich Mobile App

1.114.0 build.158

### Platform with the issue

- [X] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
volumes:
      # Do not edit the next line. If you want to change the media storage location on your system, edit the value of UPLOAD_LOCATION in the .env file
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
```

### Your .env content

```Shell
TZ=AEST
```

### Reproduction steps

```
user@host:~/docker/immich-app$ date
Fri 13 Sep 2024 10:11:27 AEST
user@host:~/docker/immich-app$ sudo docker exec -it immich_server bash
root@4be3051a10bd:/usr/src/app# date
Fri Sep 13 00:11:33 AEST 2024
```

### Additional information

I was able to fix the problem by simply NOT setting the correct timezone in `.env` and making it `TZ=Etc/UTC`.

So I think the comment in the `.env` file should be changed:
```
# To set a timezone, uncomment the next line and change Etc/UTC to a TZ identifier from this list: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List
```
, and also the `TZ=Etc/UTC` line should not be commented out by default.

**Comment by mmomjian** [comment]
`TZ=AEST` would not be a valid entry. you need to pick a "TZ identifier" from [this list](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List) - ex. `TZ=Australia/Hobart`

**Reply by drelephant** [reply]
I tried all these:
```
#TZ=AU/Sydney
#TZ="GMT+10"
#TZ=AEST
```
They all had the same issue. Only fixed by setting it to `TZ=Etc/UTC`

**Reply by mmomjian** [reply]
None of those are the correct format.

**Reply by drelephant** [reply]
Lol! Sorry, I did try `TZ=Australia/Sydney` as well.

Please tell me that's correct!

Edit: weird, I just tried that one again and it seems to have worked properly.

OK then, sorry for being silly.
